radicle/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![allow(clippy::match_like_matches_macro)]
+#![allow(clippy::explicit_auto_deref)] // TODO: This can be removed when the clippy bugs are fixed
```

Comment:
Oh, what's this? :eyes:

Reply:
There's a clippy regression -- some of the warnings from this lint are false-positives, so I had to disable it.

Reply:
Ohhh as in it wouldn't compile when you followed the clippy warning? Because I think I applied some changes for that warning but everything seemed to compile fine, so just wondering if the bug is just clippy warning you or if there's something wrong with applying the change.

Reply:
Oh, no like if I apply the change, I get a compile error.